release.md

```diff
@@ -42,21 +42,33 @@ release.
 
 ### Preparation
 
-Make sure you're on an up-to-date master branch and in cirq's root directory.
+System requirements: Linux, python3.7
+
+For MINOR / MAJOR release: Make sure you're on an up-to-date master branch and 
+in cirq's root directory.
 
 ```bash
 git checkout master
 git pull origin master  # or upstream master
 git status  # should be no pending changes
 ```
 
+For PATCH update: Make sure you're in the last version 
```

Comment:
Maybe say "in the last version's branch" to make it more clear?

Reply:
We create the release branch later. However, I could clarify: "Make sure you checked out the version you want to patch. Typically this will be something like `v${MAJOR}.${MINOR}.${LAST_PATCH}`"

Reply:
Oh, also, I think there might be a bit of misunderstanding here - I believe the tag is a better starting point than the release branch for the last version, because the branch is mutable and tags aren't (well, technically are, but shouldn't be).